Documentation/dynamic-rupture.rst

```diff
@@ -299,20 +299,44 @@ The TP parameters for which no spatial dependence has been implemented are defin
 
   &DynamicRupture
   thermalPress = 1                  ! Thermal pressurization 0: inactive; 1: active
-  IniTemp = 483.15                  ! Initial temperature [K]
-  IniPressure = -80.0e6             ! Initial pore pressure; have to be added to normal stress in your initial stress yaml file [Pa]
-  alpha_th = 1.0e-6                 ! Thermal diffusivity [m^2/s]
-  rho_c = 2.7e6                     ! Specific heat [Pa/K]
-  TP_lambda = 0.1e6                 ! Pore pressure change per unit temperature [Pa/K]
+  TP_IniTemp = 483.15                  ! Initial temperature [K]
+  TP_IniPressure = -80.0e6             ! Initial pore pressure; have to be added to normal stress in your initial stress yaml file [Pa]
+  TP_hydraulicDiffusivity = 1.0e-6     ! Thermal diffusivity [m^2/s]
```

Comment:
Hydraulic

Reply:
`parameters.par` is case-insensitive ;-)

Reply:
Haha, true :D